src/browser/rvm/rvm_message_bus.ts

```diff
@@ -415,6 +429,28 @@ export class RVMMessageBus extends EventEmitter  {
         };
         this.publish(rvmPayload, callback);
     }
+
+    public sendCoreAnalyticsEvent(event: string, payload: any): Promise<boolean> {
+        return new Promise((resolve, reject) => {
+            try {
+                const rvmMsg: CoreAnalytics = {
+                    topic: 'system',
+                    action: 'send-core-analytics-event',
+                    sourceUrl: '',
```

Comment:
I could see a world where we would want to tie analytics to a specific application, how would you recommend we handle that?

Reply:
Not sure the best solution, but I think we could pass an optional Identity param.